ui/src/app/pages/admin/user/admin-user-egress-bypass.tsx

```diff
@@ -68,22 +71,20 @@ export const AdminUserEgressBypass = (props: Props) => {
 
   const onCreateBypassRequest = () => {
     const { userId } = props;
-    setReload(false);
     egressBypassButtonDisabled = true;
     const createEgressBypassWindowRequest: CreateEgressBypassWindowRequest = {
-      startTime: (startTime as Date).valueOf(),
+      startTime: startTime.valueOf(),
       byPassDescription: bypassDescription,
     };
     setBypassDescription('');
-    setStartTime(null);
```

Comment:
clearing the description is sufficient for disabling the bypass button.  We don't need to also clear this.